components/elm/bld/namelist_files/use_cases/2015-2100_SSP585_transient.xml

```diff
@@ -32,8 +32,8 @@
 
 <!-- CMIP6 DECK compsets -->
 
-<fsurdat>lnd/clm2/surfdata_map/surfdata_ne30np4_simyr1850_2015_c171018.nc </fsurdat>
-<flanduse_timeseries>lnd/clm2/surfdata_map/landuse.timeseries_ne30np4_ssp5_rcp8.5_simyr2015-2100_c190411.nc</flanduse_timeseries>
+<fsurdat>lnd/clm2/surfdata_map/surfdata_ne30np4.pg2_simyr1850_2015_c211105.nc </fsurdat>
```

Comment:
In theory, should a simyr2015 be used for this compset (and associated use_case) that to start from 2015-01-01?

@bishtgautam , can you please also chime in?
For a hybrid SSP585 simulation (or SSP370 in a similar term) starting  using v2.LR.historical run from 2015-01-01, we will need to set `CHECK_FINIDAT_FSURDAT_CONSISTENCY       = .false.` anyway; and either fsurdat should have no impact on simulation results.

Reply:
I believe the entries in all the `.xml` files in this directory should not include any resolution-dependent values (e.g. `fsurdat`, `finidat`, `flanduse_timeseries` etc). The resolution-dependent entries should be added in `components/elm/bld/namelist_files/namelist_defaults.xml`.

Reply:
Thank you @bishtgautam  for chiming in. I don't disagree that resolution-dependent values can be consolidated into namelist_defaults.xml, as long as they can be unambiguously determined.  Here fsurdat and flanduse_timeseries values depend on compset. I am not sure if namelist_defaults.xml can recognize compset attribute, and don't see any in existing defaults.

But that can be saved for another day. Can you comment if we should use a simyr2015 surfdata in this case? Esp. when not using a hybrid run nor setting `CHECK_FINIDAT_FSURDAT_CONSISTENCY = .false.`?

Reply:
Yes they can, an example from eam  1850_CMIP6_control.xml
`<fsurdat hgrid="ne30np4" mask="oEC60to30v3" >
lnd/clm2/surfdata_map/surfdata_ne30np4_simyr1850_2015_c171018.nc </fsurdat>`

Reply:
>But that can be saved for another day.

I'm okay with adding these res-dependent values here for now, as long as, we all recognize that this compset will only correctly work with `-res ne30pg2_EC30to60E2r2`. I will defer to @rljacob if he thinks this should be fixed in the PR or in a future PR.

>Can you comment if we should use a simyr2015 surfdata in this case? Esp. when not using a hybrid run nor setting `CHECK_FINIDAT_FSURDAT_CONSISTENCY = .false.`?

I do believe if one does not use `CHECK_FINIDAT_FSURDAT_CONSISTENCY = .false.`, the fsurdat should correspond to simyr2015.

Reply:
@bishtgautam , thanks for confirming simyr2015 fsurdat should be used in general.
 
 @rljacob , I agreed variables like fsurdat, finidat can be specified in namelist_defaults.xml, using grid, mask, amd simyr modifiers. Some compset specific variables, like flanduse_timeseries, I am afraid can not be put there. For example, we have SSP370 and SSP585, they share all ( as far as lnd is concerned) except for flanduse_timeseries.

Reply:
Are we talking about the same thing?  I just searching in elm/bld/namelist_files/use_cases and find in 20thC_CMIP6_transient.xml:
`<flanduse_timeseries hgrid="ne30np4">lnd/clm2/surfdata_map/landuse.timeseries_ne30np4_hist_simyr1850_2015_c20171018.nc</flanduse_timeseries>`

That are plenty of use case that don't have an hgrid modifier for that entry but should.

Reply:
SSP370 would be a new use_case file, 2015-2100_SSP370_transient, if we stick with the current system.  Otherwise, the perl has be modified so that "ssp" can be a modifier these xml files.

Reply:
Exactly. The use_case file for SSP370 is in the master, via #4880.

Reply:
fsurdat surfdata_ne30np4.pg2_simyr1850_2015_c211105.nc is identical to the one used for v2.LR.piControl and v2.LR.historical, surfdata_ne30np4.pg2_simyr1850_c201210.nc. A new fsurdat for simyr2015 would only differ at precision level for a few land unit variables. In fact, even for simyr1850, it is using raw land unit data from around 2000. Considering these, and surfdata_ne30np4.pg2_simyr1850_2015_c211105.nc  is also in use for SSP370 simulation underway, we can keep this fsurdat for SSP585.

Attn: @golaz, @bishtgautam , @aldivi , @jjbenedict .